.github/actions/build-docs/action.yml

```diff
@@ -1,6 +1,15 @@
 name: openverse/build-docs
 description: Build Openverse full-stack docs
 
+inputs:
+  make_username:
+    required: true
+    description: The username for the GlotPress account for bulk download
+
+  make_login_password:
+    required: true
+    description: The password for the GlotPress account for bulk download
```

Comment:
Is it okay to name these the same as the env variables they get passed to? `make` has so many meanings when I was reading the usage of this action I had to do a double take to understand it.

Reply:
Updated the names to `glotpress_username` and `glotpress_password` in b2ba206.